source/mainnet/smart-contracts/onboarding-guide-ethereum-developers/faq.rst

```diff
@@ -0,0 +1,760 @@
+
+    This generates the output:
+
+    .. code-block:: console
+
+        The requested alias for address 4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JaqovViXS4j is 4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JXBDCpCaUT6
+
+    Copy your alias address to the below command.
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/<YourAliasAccountAddress>
+
+    If you insert your alias account address correctly, the command should look similar to the below line.
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JXBDCpCaUT6
+
+    **Option 5:** If you need plenty of CCD for large-scale testing,
+    please contact Concordium’s technical support via support@concordium.software.
+
+.. dropdown::  Is there something similiar to gas and tx fees?
+
+    Yes, gas is called NRG (pronounced energy) on the Concordium chain.
+    The block limit is 3 million NRG. Tx fees are fixed in Euro and are much
+    cheaper than on Ethereum.
+
+.. dropdown::  What networks can be used for testing?
+
+    Concordium has a testnet and stagenet for testing.
+
+.. dropdown:: Does Concordium have multiSig wallets?
+
+    Yes. You can create a `multiSig wallet <https://developer.concordium.software/en/mainnet/net/guides/overview-shared-accounts.html>`_ with the desktop wallet.
+    Each account has 1+ credentials, and each credential has 1+ keys.
+    You can set a threshold for the number of signers needed on an account (for example 2/3 signers).
+
+    .. note::
+
+        The desktop wallet supports all transaction types (:ref:`except smart contract transactions<transactions-overview>`).
+
+    In contrast, on Ethereum, an externally owned account cannot hold more than one key, and no externally
+    owned account multiSig wallet exist.
+    All multiSig wallets on Ethereum are smart contracts.
+
+    .. note::
+
+        You cannot import the keys from the desktop wallet to the ``concordium-client``.
+        Hence, the desktop wallet does not support smart contract transactions.
+        If you need the multiSig wallet to manage a smart contract,
+        you need to deploy a smart contract multiSig wallet
+        (similar to how Ethereum uses multiSig wallets at work).
+
+.. dropdown::  What does `invoke` mean?
+
+    `Invoke` may refer to:
+        - It can mean to execute or initiate a function. It is equivalent to Ethereum saying: "Calling a smart contract function".
+
+        - In the context of the ``concordium-client`` tool, it means to simulate a tx locally on your node via the `invoke` command of the ``concordium-client`` tool instead of sending the tx to the blockchain network and executing it on-chain. Since the tx was simulated it was not inserted by the bakers in a block and is not part of the blockchain and state changes that the `invoke` command makes are discarded afterwards.
```

Comment:
```suggestion
        - In the context of the ``concordium-client`` tool, it means to simulate a tx locally on your node via the `invoke` command of the ``concordium-client`` tool instead of sending the transaction to the blockchain network and executing it on-chain. Since the transaction was simulated it was not inserted by the bakers in a block and is not part of the blockchain and state changes that the `invoke` command makes are discarded afterwards.
```